apps/src/music/player/sequencer/Simple2Sequencer.ts

```diff
@@ -102,13 +108,21 @@ export default class Simple2Sequencer extends Sequencer {
   }
 
   // Beginning of a play_together block.
-  playTogether() {
+  // The isBlock parameter is true when this is called by the play_together block.
+  playTogether(isBlock: boolean) {
     const nextMeasure = this.getCurrentMeasure();
     this.sequenceStack.push({
       measure: nextMeasure,
       together: true,
       lastMeasures: [nextMeasure],
     });
+
+    if (isBlock) {
+      this.validator.setSatisfiedCondition(
```

Comment:
I feel a little weird about sequencing logic needing to know about validation code. Could we instead provide a `getWorkspaceBlocks()` or similar function to the MusicValidator (like getPlaybackEvents()) and check there? We'd also be able to validate against more blocks then.

Reply:
Fair point.  My reasoning was that we are seeing students with stray blocks in their workspace that don't actually impact their song, and I wanted to only pass validation when a block was actually executed on playback.  But now I'm realizing that our execution model runs upfront, so this isn't the right approach anyway.  Let me dig into this some more and revisit.

Reply:
I've removed that logic for now, along with this example validation: `{"name": "used_block", "value": "play_together"}`.